src/main/java/com/pmm/budgetcalculator/expensing/domain/expense/ExpenseFactory.java

```diff
@@ -0,0 +1,13 @@
+package com.pmm.budgetcalculator.expensing.domain.expense;
+
+import com.pmm.budgetcalculator.foundation.AbstractFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ExpenseFactory implements AbstractFactory<Expense> {
```

Comment:
Read about abstract factory pattern, factory method pattern and builder pattern and decide on one approach. You can also use simple Constructors or empty constructors with setting every needed value by setter. Your call.